src/parser/shared/modules/SpellUsable.test.js

```diff
@@ -231,6 +231,7 @@ describe('core/Modules/SpellUsable', () => {
         },
         targetIsFriendly: true,
         maxCharges: 1,
+        name: undefined,
```

Comment:
what is this?

Reply:
This test was failing because the prop `name` was set to undefined by the calling code, but not present here in the expected value.

However, this test is also not run on CI (pretty sure) so it was only failing locally. Just an annoyance. Would be better to clean up the `SpellUsable` tests so they don't check literally every prop but that would require me sitting down and spending time to understand what they are actually trying to test.